sklearn/metrics/_classification.py

```diff
@@ -1321,6 +1321,16 @@ def fbeta_score(
     Asymptotically, `beta -> +inf` considers only recall, and `beta -> 0`
     only precision.
 
+    The formula for F-beta score is:
+
+    .. math::
+
+       F_\\beta = \\frac{(1 + \\beta^2) \\text{tp}}
+                        {(1 + \\beta^2) \\text{tp} + \\text{fp} + \\beta^2 \\text{fn}}
+
+    Where :math:`\\text{TP}` is the number of true positives, :math:`\\text{FN}` is the
+    number of false negatives, and :math:`\\text{FP}` is the number of false positives.
```

Comment:
Given that the formula itself uses lowercase characters:

```suggestion
    Where :math:`\\text{tp}` is the number of true positives, :math:`\\text{fn}` is the
    number of false negatives, and :math:`\\text{fp}` is the number of false positives.
```

Reply:
Ah thanks, I got mixed up because we used capitals in the `f1` docstring (from #27936 - though a mix of capitals and lower case is used throughout that PR). Fixed!